sycl/test/esimd/regression/windows_build_test.cpp

```diff
@@ -6,36 +6,42 @@
 //
 //===----------------------------------------------------------------------===//
 // REQUIRES: windows
-// RUN: %clangxx -fsycl -fsycl-device-only -fsyntax-only -Xclang -verify %s -I %sycl_include
+// RUN: %clangxx -fsycl -fsyntax-only -Xclang -verify %s -I %sycl_include
 // expected-no-diagnostics
 
 // The tests validates an ability to build ESIMD code on windows platform
 
-#include <iostream>
 #include <CL/sycl.hpp>
+#include <iostream>
 #include <sycl/ext/intel/esimd.hpp>
+#include <sycl/ext/intel/esimd/xmx/dpas.hpp>

```

Comment:
`#include <sycl/ext/intel/esimd.hpp>` should be the only ESIMD header included in the user code. Doesn't this work for this test?